docker-compose-dev.yml

```diff
@@ -1,136 +1,79 @@
-# Example Wikibase docker-compose setup
 version: '3.4'
 
 x-common-variables: &wikibase_variables
-  DB_SERVER: mysql.svc:3306
-  MW_ADMIN_NAME: ${MW_ADMIN_NAME}
-  MW_ADMIN_PASS: ${MW_ADMIN_PASS}
-  MW_ADMIN_EMAIL: ${MW_ADMIN_EMAIL}
-  MW_WG_SECRET_KEY: ${MW_SECRET_KEY}
-  # Disable jobs running after requests when wikibase_jobrunner is defined
-  MW_WG_JOB_RUN_RATE: 0
-  DB_USER: ${DB_USER}
-  DB_PASS: ${DB_PASS}
-  DB_NAME: ${DB_NAME}
-  WIKIBASE_HOST: ${WIKIBASE_HOST}
-  QS_PUBLIC_SCHEME_HOST_AND_PORT: http://localhost:8840
-
-x-common-variables: &wikibase_extra_variables
-  MW_ELASTIC_HOST: ${MW_ELASTIC_HOST}
-  MW_ELASTIC_PORT: ${MW_ELASTIC_PORT}
-
+    WIKIBASE_HOST: localhost
```

Comment:
why are these variables not in env file?

Reply:
Let's talk about this during the daily.